frontends/election-manager/src/screens/write_ins_screen.test.tsx

```diff
@@ -17,6 +17,34 @@ test('No CVRs loaded', async () => {
   expect(screen.queryByText('Transcribe')).not.toBeInTheDocument();
 });
 
+test('Tally results already marked as official', async () => {
+  const backend = new ElectionManagerStoreMemoryBackend({
+    electionDefinition,
+  });
+
+  await backend.addCastVoteRecordFile(
+    new File([abbreviatedCvrData], 'cvrs.jsonl')
+  );
+
+  renderInAppContext(<WriteInsScreen />, {
+    backend,
+    electionDefinition,
+    isOfficialResults: true,
+  });
+
+  await screen.findByText(/No further changes may be made/);
+
+  const transcribeButtons = await screen.findAllByText(/Transcribe \d/);
```

Comment:
Nit: I think you can use `screen.getAllByText` here, since you don't need to wait for any async DOM updates. In general, I try to only use the async testing-library queries when needed in order to indicate to readers that something is updating in the DOM, but that's just a personal style choice so take it or leave it

Reply:
Ah, thanks - I like that as a guiding principle. I also need to get more familiar with the react testing tools in general, so tips very welcome